Sources/PackageCollectionsSigning/PackageCollectionSigning.swift

```diff
@@ -95,7 +95,7 @@ public struct PackageCollectionSigning: PackageCollectionSigner, PackageCollecti
 
     private let observabilityScope: ObservabilityScope
 
-    public init(trustedRootCertsDir: URL? = nil, additionalTrustedRootCerts: [String]? = nil, observabilityScope: ObservabilityScope, callbackQueue: DispatchQueue) {
+    public init(trustedRootCertsDir: URL? = nil, additionalTrustedRootCerts: [String]? = nil, observabilityScope: ObservabilityScope = ObservabilitySystem { _, _ in }.topScope, callbackQueue: DispatchQueue) {
```

Comment:
package-collection-signer can do it: https://github.com/apple/swift-package-collection-generator/blob/main/Sources/PackageCollectionSigner/PackageCollectionSign.swift#L84

I assume others can too?

Reply:
> package-collection-signer can do it: https://github.com/apple/swift-package-collection-generator/blob/main/Sources/PackageCollectionSigner/PackageCollectionSign.swift#L84
> 
> I assume others can too?

That's a good point, let me check tomorrow why I failed to to the same 🤔

Reply:
afaik this should be possible

Reply:
Silly me, it does indeed work. I suspect what happened was that the original error of `Cannot find 'ObservabilitySystem' in scope` turned into `Argument 'observabilityScope' must precede argument 'callbackQueue'` while trying to find where this is imported from because I had the `sign` call wrong. 

I only noticed this now as I was re-doing everything bit by bit mirroring the Collection Generator.

Although I'm pretty sure I'd copied the `sign` call from the Collection Generator but who knows 😅